src/layouts/default/PlayerOSD/PlayerFullscreen.vue

```diff
@@ -121,7 +121,15 @@
                 store.curQueueItem?.streamdetails?.stream_title
               "
             >
-              <h4 class="fullscreen-track-info-subtitle">
+              <!-- radio live metadata -->
+              <h4
+                v-if="store.curQueueItem.streamdetails.stream_title"
```

Comment:
```suggestion
                v-if="store.curQueueItem.streamdetails.stream_title && store.curQueueItem.streamdetails.stream_title.includes(' - ')"
```

Reply:
For now just do a check if the hyphen is present as that is in 90% of the cases true when artist+title is in streamtitle. Not waterproof but at least as bit of guard .

Reply:
Ok I've added this guard.